woocommerce/woocommerce
# Fix TT3 styles and typography in the single product's attributes table

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Similar to:
https://github.com/woocommerce/woocommerce/pull/37639/files

This PR introduces styling improvements in the attributes table of the "Additional Information" product tab when viewing the single product template.

*Before*:
![Before](https://cdn-std.droplr.net/files/acc_1208648/kifK5O)

*After*:
![After](https://cdn-std.droplr.net/files/acc_1208648/wVBKlt)

### How to test the changes in this Pull Request:
Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create or use an existing site with TT3 (or Zaino)
2. Ensure you have at least one product with multiple attribute values.
3. Visit the single product's page and notice the styling issues in the attributes table in the Additional Information tab.
4. Apply this patch, _hard-refresh_ for CSS caching issues, and ensure the attributes table looks nice!
